ext/libgav1_android.sh

```diff
@@ -0,0 +1,41 @@
+# If you want to use a local build of libgav1 to be used with the android NDK,
+# you must clone the libgav1 repo in this directory first, then enable CMake's
+# AVIF_CODEC_LIBGAV1 and AVIF_LOCAL_LIBGAV1 options.
+#
+# This script will build libgav1 for the default ABI targets supported by
+# android. You must pass the path to the android NDK as a parameter to this
+# script.
+#
+# Android NDK: https://developer.android.com/ndk/downloads
+#
+# The git SHA below is known to work, and will occasionally be updated. Feel
+# free to use a more recent commit.
+#
+# cmake and make must be in your PATH.
+
+if [ $# -ne 1 ]; then
+  echo "Usage: ${0} <path_to_android_ndk>"
+else
```

Comment:
Nit: After the 'echo' statement, should the script just do "exit 1"? Then we won't need the "else".

Reply:
Done.